frontend/src/views/DoctorOrders/OrderCard/OrderCard.test.tsx

```diff
@@ -11,7 +11,7 @@ const doctorOrders = [
     doctorID: '1122334455',
     doctorName: 'Dr. Jane Doe',
     drugNames: 'Turalio 200 MG Oral Capsule',
-    drugNdcCode: '65597-402-20',
```

Comment:
Is this the only change that was needed? There wasn't another lookup table needing to be added or updated?

Reply:
separate PR: github.com/mcode/pims/pull/130